python/cudf/cudf/tests/test_orc.py

```diff
@@ -783,6 +786,28 @@ def test_orc_writer_decimal(tmpdir, scale):
     assert_eq(expected.to_pandas()["dec_val"], got["dec_val"])
 
 
+def test_orc_reader_multiple_files(datadir):
+
+    path = datadir / "TestOrcFile.testSnappy.orc"
```

Comment:
Might need to add a file with multiple stripes and choose different ones in each set to make whole.